pkg/config/config.go

```diff
@@ -855,3 +856,28 @@ func applyOverrides(config Config) {
 		config.Set(k, v)
 	}
 }
+
+// TracemallocEnabled is a helper to get the effective tracemalloc
+// configuration.
+func TracemallocEnabled(config Config) bool {
+	pyVersion := config.GetString("python_version")
+	wTracemalloc := config.GetBool("tracemalloc_debug")
+	if pyVersion == "2" && wTracemalloc {
+		wTracemalloc = false
+	}
+
+	return wTracemalloc
+}
+
+// GetNumWorkers is a helper to get the number of workers for
+// a given config.
+func GetNumWorkers(config Config) int {
```

Comment:
I think we should overwrite `check_runners` value in the post processing of the configuration (like for secrets or proxy).
This would avoid have a `config.GetInt('check_runners')` somewhere by mistake and would also keep the flare and the `configcheck` command in sync with the agent behavior.

Also adding a small comment on why we force only 1 runner when using tracemalloc could be useful.